tests/h/search/config_test.py

```diff
@@ -168,22 +181,58 @@ def test_updates_index_atomically(self, client):
         }
 
         update_aliased_index(client, 'new-target')
-
         client.conn.indices.update_aliases.assert_called_once_with(body={
             'actions': [
                 {'add': {'index': 'new-target', 'alias': 'foo'}},
                 {'remove': {'index': 'old-target', 'alias': 'foo'}},
             ],
         })
 
-    def test_raises_if_called_for_concrete_index(self, client):
+    @pytest.mark.parametrize(
+        'error',
+        ((elasticsearch1.NotFoundError),
+         (elasticsearch.NotFoundError))
+    )
+    def test_raises_if_called_for_concrete_index(self, client, error):
         """Raise if called for a concrete index."""
-        client.conn.indices.get_alias.side_effect = NotFoundError('test', 'test desc')
+        client.conn.indices.get_alias.side_effect = error('test', 'test desc')
 
         with pytest.raises(RuntimeError):
             update_aliased_index(client, 'new-target')
 
 
+class TestUpdateIndexSettings(object):
+    # todo: add a test for positive behavior
```

Comment:
Note this test class didn't exist before at all. I would like to add another test here for when everything goes right to verify the correct mapping gets used but I ran out of time and wanted to get your eyes on this again.